servers/user-list-search/src/datasource/elasticsearch/elasticsearchSearch.integration.ts

```diff
@@ -317,7 +316,7 @@ describe('Elasticsearch Search Query', () => {
     expect(response.edges[1].node.savedItem.id).toBe(333);
   }, 10000);
 
-  it('should return search response with paginated fields first', async () => {
+  it('should return search response with paginated fields: first', async () => {
```

Comment:
I examined the response from the search method and corrected this test. It's possible that the scoring algorithm is slightly different.